### PR TITLE
bump to bionic; gcc-9 and clang-10

### PR DESCRIPTION
**Pull request by psi29a**
Let's see what Travis-CI says...

### CODE REVIEW

**Comment by psi29a** [comment]
And we're in bionic... plus sdl2 2.0.10 from our ppa.

File: .travis.yml [hunk 1]
```diff
@@ -42,45 +41,45 @@ matrix:
       os: osx
       osx_image: xcode10.2
       if: branch != coverity_scan
-    - name: OpenMW (all) on Ubuntu Xenial GCC-5
+    - name: OpenMW (all) on Ubuntu Xenial GCC-7
```
Comment (Mingun) [inline .travis.yml]: You forgot to change name to Bionic

Comment (psi29a) [reply .travis.yml]: Derp

Comment (psi29a) [reply .travis.yml]: #2735 2735

**Pull request merged**